assessment/models.py

```diff
@@ -397,7 +397,7 @@ def get_task_generator(self):
         return task_generator
 
     def is_active(self) -> bool:
-        return self.start_date_time <= datetime.datetime.now(datetime.timezone.utc)
+        return self.start_date_time <= datetime.datetime.now(datetime.timezone.utc) <= self.get_event_end_date_time()
```

Comment:
Hello, Jo. Allow me to ask, is the end time for an event is derived?

Reply:
Hello, Alex. For now it is still derived. However, I am planning of introducing a new attribute that will store the end time on every test flow tool insertion. Nevertheless, a conversion to UTC dateetime is still required on the comparison.

Regards,
Johanes Christian

Reply:
Noted, Jo. Thanks for the clarification.